tests/api_tests/test_questionnaire_response_api.py

```diff
@@ -1707,6 +1707,70 @@ def test_questionnaire_life_functioning_survey(self):
         self.assertEqual(summary['questionnaireOnLifeFunctioningAuthored'], '2022-09-06T14:32:28')
         self.assertEqual(summary['questionnaireOnLifeFunctioningTime'], '2022-09-07T01:02:03')
 
+    def test_remote_identity_verified(self):
+        """ Test to see if a remote ID verification True Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
```

Comment:
Shouldn't need this for the tests, you can do the POST with the default account.

Reply:
I see, thanks for catching this!